data/ita/Richardson_2001_Fortunio-Regole/metadata.txt

```diff
@@ -0,0 +1,25 @@
+publicationType : "book"
+publicationTitle : "Regole grammaticali della volgar lingua"
+publicationDate : "2001"
+publicationPlace : "Roma/Padova"
+publicationSeries : ""
+publicationNum: ""
+publicationStemmaNum: ""
+publicationAuthors : "Richardson, Brian"
+publicationPage : ""
+publicationLink : ""
+workTitle : "Regole grammaticali della volgar lingua"
+workViaf : "189376359"
+workOrigDate : "1516"
+workOrigPlace : "Trieste?"
+workAuthor : "Giovanni Francesco Fortunio"
+workAuthorViaf : "76535047"
+workGenre : "grammar"
+workLangCode : "ita"
+stemmaType : "observed"
+contam : "no"
+extraStemmContam : "no"
+rootType : "original"
+contributor : "Jean-Baptiste Camps"
+contributorORCID : "0000-0003-0385-7037"
+note : "Stemma encoded not from the edition, but from Trovato (2004), Sugli stemmi bipartiti"
```

Comment:
Should we include the years of the manuscripts here as we already have them?

Reply:
I'm too lazy right now to do it, but it could be a welcomed addition !